### ISSUE TITLE
Refactoring and Find Usages do not work with @Accessors(fluent = true)

### ISSUE THREAD

**Issue by xnuter** [mplushnikov/lombok-intellij-plugin#487]
### Short description
I have a class like this:

```
@Value
    @ToString
    @Accessors(fluent = true)
    @Builder
    class DataItem {
        @NonNull String name;
        @NonNull Instant createTimestamp;
        @NonNull String content;
    }
```

and then Find Usages for fields doesn't give anything.  

### Expected behavior
Find Usages will show reads (e.g. `dataItem.name()`) and writes (e.g. `DataItem.builder().name()`).

### Version information

 * **IDEA Version:** 2017.3.5
 * **JDK Version:** JDK 1.8.0_171
 * **OS Type & Version:** MacOS 10.12.6
 * **Lombok Plugin Version:** 0.16-2017.3.3
 * **Lombok Dependency Version:** 1.16.12

### Steps to reproduce

 1. Create a Java class as in the example above
 2. Construct an instance and call its getters
 3. Try to find usages of a field (e.g. `name`)

E.g. 

```
DataItem item = DataItem.builder()
                        .name("a")
                        .createTimestamp(Instant.now())
                        .content("content")
                        .build();
System.out.println("Item.name = " + item.name());
```

### Stacktrace
No error happens. Just silently shows nothing.

**Comment by xnuter** [comment]
Updated IntelliJ to 2018.1 and the Lombok plugin to 0.17-2018.1 still no success.

It highlights fields as if they are used somewhere, but it doesn't show where.

**Comment by mplushnikov** [comment]
Closed as duplicate of #186
